lib/localization/arb/intl_tr.arb

```diff
@@ -0,0 +1,325 @@
+{
+  "playback": "Geri çalma",
+  "@playback": {
+    "description": "Bildirim kanalı adı olarak kullanılır"
+  },
+  "playbackControls": "Geri çalma kontrolleri",
+  "@playbackControls": {
+    "description": "Bildirim kanalı açıklaması olarak kullanılır"
+  },
+  "play": "Oynat",
+  "pause": "Duraklat",
+  "stop": "Durdur",
+  "next": "İleri",
+  "previous": "Önceki",
+  "loopOff": "Döngüden çık",
+  "loopOn": "Döngüye gir",
+  "search": "Ara",
+  "shuffleAll": "Tümünü karıştır",
+  "playRecent": "En son oynatılanı çal",
+  "artistUnknown": "Bilinmeyen sanatçı",
+  "@artistUnknown": {
+    "description": "Bilinmeyen sanatçı için etiket"
+  },
+  "track": "Parça",
+  "tracks": "Parçalar",
+  "tracksPlural": "{count, plural, =1{{count} track} other{{count} tracks}}",
```

Comment:
Could you also translate this one?
Here you need to translate the words "track" and "tracks"